pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -812,7 +813,14 @@ private bool ParseReintegrateExcludeFeature(NameValuePair pair)
                 return false;
             }
 
-            ExcludeFeatures.Add(calc);
+            if (pair.Name.Equals(ARG_MPROPHET_FEATURES_MPROPHET_EXCLUDE_SCORES.Name))
```

Comment:
Better to add the IList as a parameter than to do this comparison with this if clause.